drivers/mil_passive_sonar/nodes/sonar_test.py

```diff
@@ -10,6 +10,9 @@
 import sys
 
 
+# This should be turned into a unit test
```

Comment:
Agreed, I separated the sonar stuff from the great merge stuff and will add tests before merging.